subiquity/server/controllers/filesystem.py

```diff
@@ -404,6 +404,7 @@ def disallowed_encryption(msg):
         return info
 
     async def _examine_systems(self):
+        self._variation_info.clear()
```

Comment:
do we also need to reset `self._info`?

Reply:
That's a fair point. `self._info` is set through a call to `set_info_for_capability` - which only happens after a POST to /storage/guided (or /storage). I would need a build a test-case for that as well.

Reply:
OK, not blocking for this